app-frontend/src/app/components/lab/inputNode/inputNode.html

```diff
@@ -21,10 +21,16 @@
     <div ng-if="!$ctrl.bands.length && $ctrl.fetchingDatasources">
       Loading band data
     </div>
-    <div ng-if="$ctrl.selectedProject && !$ctrl.bands.length && !$ctrl.fetchingDatasources">
-      <a class="btn btn-flat btn-block" ui-sref="imports.datasources.detail({
-        datasourceid: $ctrl.firstDatasourceWithoutBands().id
-      })">Requires defined bands</a>
+    <div ng-if="$ctrl.selectedProject && !$ctrl.bands.length && !$ctrl.fetchingDatasources"
+         class="form-group search-form">
+      <input class="form-control" type="number" min="0"
```

Comment:
Specifying `min="0"` does prevent you from applying negative band index using the up and down button, but this could be easily bypassed by manually entering a negative number as I tested. We probably want to do something in the `onBandChange` function in addition to prevent it.